src/canister/individual_user_template/src/api/post/send_update_post_cache.rs

```diff
@@ -37,11 +37,18 @@ pub fn send_update_post_cache(post_id: &u64) {
     });
 
     if home_feed_index_score_item.is_some() {
+        let home_feed_index_score_clone = home_feed_index_score_item.clone();
         let _ = call::notify(
             post_cache_canister_principal_id,
             "update_post_home_feed",
+            (home_feed_index_score_clone.unwrap(),),
+        );
+        let _ = call::notify(
```

Comment:
should this change be in home_feed_index_score_item or hot_or_not_index_score_item. Once check how the scores are being calculated and which score do we want to keep. I think it should be HotOrNot score but once check and let's discuss.

Reply:
might have to implement recalculate_yral_feed_score, but have to check